src/tests/running.md

```diff
@@ -59,6 +59,43 @@ Likewise, you can test a single file by passing its path:
 ./x.py test src/test/ui/const-generics/const-test.rs
 ```
 
+### Run the UI test suite
+
+The UI test suite is special in that it tests the command-line output
+of the compiler, e.g. the wording and formatting of warnings and errors.
+Every UI test file such as `foo.rs` will have a corresponding file
+`foo.stderr` describing its expected output.
+
+Similar to any other test suite, the UI test suite can be run with:
+
+```bash
+./x.py test src/test/ui
+```
+
+If you have made a change to the way that the compiler formats its output,
+it would be extraordinarily tedious to have to reformat every `*.stderr`
+file manually. Fortunately there exists a flag that will allow you to
+"bless" the current output of the compiler as the expected output.
+When using this flag with the test runner, any UI tests that would
+ordinarily fail will instead have their `*.stderr` files overwritten
+with whatever output the compiler produces:
+
+```bash
+./x.py test src/test/ui --bless
```

Comment:
Most of this info is already present in the '## Editing and updating the reference files' section below. Can you move it to that section so it's not duplicated?